src/sphinx_2i2c_theme/assets/styles/sphinx-2i2c-theme.scss

```diff
@@ -1,123 +1,4 @@
 @import "../sphinx-book-theme.css";
-
-/* Overriding Book Theme colors to standardize for 2i2c */
-:root {
-  --pst-font-size-base: none;
-  --pst-color-primary: 87, 154, 202;
-  --pst-color-secondary: 208, 102, 129;
-  --pst-color-navbar-link-active: var(--pst-color-primary);
-  --pst-color-sidebar-link-active: var(--pst-color-primary);
-  --pst-color-link: var(--pst-color-primary);
-  --navbar-height: 3em;
-  --logo-bold: 700
-}
-
-// Navigation bar on top
-nav.nav-2i2c {
-  border-bottom: 1px solid rgba(0, 0, 0, 0.1);
-  position: sticky;
-  top: 0;
-  left: 0;
-  background-color: white;
-  z-index: 999;
-  height: var(--navbar-height);
-  align-items: center;
-  padding: 0;
-
-  a {
-    color: rgb(var(--pst-color-primary));
-  }
-
-  ul {
-    margin: 0;
-    padding-left: 1em;
-  }
-
-  li {
-    padding: 0 1em;
-    list-style: none;
-  }
-
-  .nav-logo {
-    padding: 0;
-    /* Max width of the left navbar in the book theme */
-    max-width: 275px; 
-  }
-
-  button:hover, .nav-2i2c a:hover {
-    color: rgb(var(--pst-color-primary));
-    text-decoration: none;
-  }
-
-  #topbar-left {
-    p.nav-title-text {
-      margin: 0;
-      width: 275px;
-    }
-
-    > a {
-      font-size: 1.6em;
-      font-family: "Fira Sans";
-      font-weight: var(--logo-bold);
-      margin-top: .3em;
-      text-align: center;
-      text-decoration: none;
-  
-      span.nav-subtitle-text {
-        font-weight: 400;
-        color: rgb(var(--pst-color-primary));
-        padding-left: .2em;
-      }
-    }
-  }
-  
-  #topbar-middle > *:first-child {
-    /* First link doesn't need left padding */
-    padding-left: 0 !important;
-  }
-
-  #topbar-right {
-    #support-button-2i2c {
-      font-size: 1.1em;
-      padding: .15em 0 .3em 0;
-      border: 1px rgb(var(--pst-color-primary)) solid;
-      border-radius: 0.5em;
-      transition: background-color .2s;
-      text-decoration: none;
-      &:hover {
-        background-color: rgb(247, 247, 247);
-      }
-    }
-  }
-}
-
-// Modify book theme site navigation to accommodate the topbar
-#site-navigation {
-  height: calc(100vh - var(--navbar-height)) !important;
-  top: var(--navbar-height) !important;
-
-  .navbar-brand-box, #site-navigation .navbar-brand-box .navbar-brand {
-    padding-top: 0;
-  }
-
-  h1#site-title {
-    font-family: "Fira Sans";
-    font-size: 1.1em !important;
-    color: rgb(var(--pst-color-primary));
-    font-weight: 500;
-  }
-}
-
-.fixed-top {
-  top: var(--navbar-height);
-}
-
-
-div.dropdown-menu i {
-  padding-left: .5em;
-}
-
-/* Topbar of main theme */
-.topbar {
-  z-index: 800;
-}
+@import "abstract/variables";
+@import "sections/header";
+@import "sections/sidebar-primary";
```

Comment:
I really like this pattern of importing specific (defined per component) scss files!